src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php

```diff
@@ -68,18 +71,20 @@ public function __construct(
      *
      * @param AMQPMessage $message A filer message.
      *
-     * @return boolean True if success, false otherwise.
+     * @return integer
      */
     public function execute(AMQPMessage $message)
     {
         // @codingStandardsIgnoreStart
         $routingKey = $message->delivery_info['routing_key'];
 
+        $msgStatus = self::MSG_ACK;
```

Comment:
Aren't these states ConsumerInterface::MSG_ACK ?

See: https://github.com/php-amqplib/RabbitMqBundle/blob/master/RabbitMq/ConsumerInterface.php

Reply:
Yeah, these states are from ConsumerInterface and DoiConsumer implements this interface